compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -82,6 +82,7 @@ class ProtobufGenerator(val params: GeneratorParams) extends DescriptorPimps {
       .add(s"def isEmpty: Boolean = false")
       .add(s"def isDefined: Boolean = true")
       .add(s"def number: Int")
+      .add(s"def getValue: Option[scala.Any] = None")
```

Comment:
Can you rename `getValue` to `valueOption` (like the standard `headOption` and `lastOption`), the reason is that in ScalaPB usually `x` returns an `Option[X]` and `getX` returns an `X`, so `getValue` would be inconsistent with that.